### PR TITLE
filtering small ui update fix

### LINKED ISSUE #11957: [collection filters] Tooltip for filtering rule enable/disable button does not track button state
**To Reproduce**

1. Launch darktable and add some filtering rules that are not pinned to the toolbar (pinning actually deactivate enable/disable button as pinned rule can't be disabled).You may disable some added rules and relaunch darktable to see that the issue does not depend on the initial state of the buttons.
2. Hover mouse pointer over enable/disable button, you should see correct information: "this rule is enabled" or "this rule is disabled" depending on the state of the rule.
3. Press the button to change the state of the rule, the tooltip will not change.

- darktable version: 3.9.0+1672~g8d699d714

**Expected behavior**
Tooltip should correctly show current rule state.

### PR DESCRIPTION
**Pull request by AlicVB**
this fix #11957 #11959

### CODE REVIEW

File: src/dtgtk/range.c [hunk 1]
```diff
@@ -483,6 +483,10 @@ static void _popup_date_update(GtkDarktableRangeSelect *range, GtkWidget *w)
 
   pop->internal_change++;
 
+  dt_bauhaus_combobox_clear(pop->type);
+  dt_bauhaus_combobox_add(pop->type, _("fixed"));
+  if(w == range->entry_min || w == range->entry_max) dt_bauhaus_combobox_add(pop->type, _("relative"));
```
Comment (TurboGit) [inline src/dtgtk/range.c:488]: In this case I think we can improve a bit more. If a single entry on the combo it seems strange to have it active and when clicking on it see only a single value. I would therefore make the widget non active when a single value is in the combo.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Possible simple enhancement.

File: src/dtgtk/range.c (hunk 1 above)
Comment (AlicVB) [reply src/dtgtk/range.c:488]: done

**Review by TurboGit — APPROVED** [review]
Works for me, thanks!

**Pull request merged**